packages/sdk/src/HopBridge.ts

```diff
@@ -1994,15 +2008,17 @@ class HopBridge extends Base {
     }
     recipient = checksumAddress(recipient)
 
-    let ammWrapper = await this.getAmmWrapper(sourceChain, sourceChain.provider)
+    const ammWrapper = await this.getAmmWrapper(sourceChain, sourceChain.provider)
+    const l2Bridge = await this.getL2Bridge(sourceChain, sourceChain.provider)
+    const attemptSwapAtSource = this.doesUseAmm && this.shouldAttemptSwap(amountOutMin, deadline)
```

Comment:
same comment as above

```suggestion
    const attemptSwapAtSource = this.shouldAttemptSwap(amountOutMin, deadline)
```

Reply:
Resolved with 56d91bae233fc3fdecafbecb07dac35a5db04144